### ISSUE TITLE
Anyone have a solution for the weather fix ?

### ISSUE THREAD

**Issue by FeaReDGlock** [MarcoPixel/SysDash#54]
Hi,
Anyone have a solution for the weather fix of http://wxdata.weather.com/ ?

Because i'm looking to still having Sysdash but i want the weather too :)

Thank you 💯

**Comment by BoscoDomingo** [comment]
I too am interested in this. I'm gonna assume the weather.com API changed or something?

**Comment by zhaobenny** [comment]
```
[Rainmeter]
Group=SysDash | Weather
Update=10000
AccurateText=1
BackgroundMode=2
SolidColor=0,0,0,1

; = IMPORTANT =
; = For configuring the rainmeter skin, right-click the visualizer and click on "Open settings".
; = You can also open the variables.ini file located in:
; = "My Documents\Rainmeter\Skins\SysDash\@Resources"

; Small context menu when you right-click the skin
ContextTitle=" Open settings"
ContextAction=[!ActivateConfig "#ROOTCONFIG#\Settings" "general.ini"]
ContextTitle2=" Open variables file"
ContextAction2=["#@#variables.ini"]

[Metadata]
Name=SysDash Dashboard
Author=marcopixel
License=MIT License
Information=An minimalistic, still stylish dashboard-like skin with modular components.

[Variables]
; Includes the variables used for the skin.
@include=#@#variables.ini
@include2=#@#include\MeterStyles.inc

; Measure to get the weather data and properly parse it
[MeasureWeather]
Measure=Plugin
Plugin=WebParser
Url=https://en.wttr.in/#WeatherLocation#?0nQT&format=j1
RegExp="(?siU)"temp_#WeatherTemperatureUnit#": "(.*)".*"weatherCode": "(.*)".*"value": "(.*)"."
UpdateRate=1600
FinishAction=[!Update] [!Update]

[MeasureCurrentCity]
Measure=String
String = #WeatherLocation#
Substitute="":"na"

[MeasureCurrentCode]
Measure=Plugin
Plugin=WebParser
URL=[MeasureWeather]
StringIndex=2
Substitute="":"na"

[MeasureCurrentTemp]
Measure=Plugin
Plugin=WebParser
URL=[MeasureWeather]
StringIndex=1
Substitute="":"N/A"

[MeasureCurrentDesc]
Measure=Plugin
Plugin=WebParser
URL=[MeasureWeather]
StringIndex=3
Substitute="":"na"

[MeterWeatherIcon]
Meter=Image
MeasureName=MeasureCurrentCode
Path=#@#images\weather
X=(#Margin#*#Scale#)
Y=(20*#Scale#)
W=(40*#Scale#)
H=(40*#Scale#)
Padding=0,0,0,(20*#Scale#)
ImageTint=#MainColor#
ImageCrop=-30,-30,61,61,5

[MeterWeatherTempText]
Meter=String
MeasureName=MeasureCurrentTemp
MeterStyle=StyleValue
X=(15*#Scale#)R
Y=((80/2)*#Scale#)
Postfix=[\x00B0]

[MeterWeatherCityText]
Meter=String
MeasureName=MeasureCurrentCity
MeterStyle=StyleSecondary
X=((#Width#+#Margin#)*#Scale#)
Y=(30*#Scale#)
InlineSetting4=Color | #FontColor#,255
[MeterWeatherDescText]
Meter=String
MeasureName=MeasureCurrentDesc
MeterStyle=StyleSecondary
X=0r
Y=0R
```

**The weather icons do not work properly yet**, you have to rename the weather icons in @Resources/images/weather to [these](https://github.com/chubin/wttr.in/blob/master/lib/constants.py) or download my imperfect interpretation of the icons to weather code [if you're lazy.](https://mega.nz/#!DVZQHSTI!HlT8y7tIdpl6zK2pMP-dZ8BbW-My7sX6RWO0TBYCMLc)
(Can't figure out how to make the night/daytime weather icons work)

**Comment by marcopixel** [comment]
Will be fixed in the future, check #56 for more information.
